### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites- Edit_Screen Reader: User is unable to Rename the Name in the cell when Narrator is ON

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#162]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 
Screen Reader: Narrator
**MAS ID Violated:** MAS1.3.2, MAS2.1.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Sites" in the left menu section and Press Enter.
5) Tab till "More" commands button of the existing website inside the grid and Press Enter.
6) Tab till "Edit" in the expanded menu and Press Enter.
7) Tab till "Settings" tab item and press Enter.
8) Tab till "Select Folder" and expand it.
9) Tab till "Go Up" button and press Enter
10) Navigate to "More" button and select "Rename"
11) Start Screen Reader (Narrator/NVDA).
12) Verify user is able to Rename the name in the cell 

**Actual Result:**
User is unable to Rename the Name in the cell when Narrator is ON

**Note:** Similar issue  observed through Keyboard as well

**Expected Result:**
User should be able to Rename the Name when narrator is ON

**User Impact:** 
Screen Reader users or Visually Impaired Users won't be able to edit the Name and this can create inconvenience to the user

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachments:**
[Bug_162_User is unable to rename the name.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331592/Bug_162_User.is.unable.to.rename.the.name.pptx)

**Comment by jhkimnew** [comment]
If the rename menu is selected with space bar, we can rename the folder.
Considering there is a work-around way, this should be closed.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Edge; #Narrator; #MAS2.1.1; #Keyboard

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
